addon/components/listing.js

```diff
@@ -146,6 +152,77 @@ export default class ListingComponent extends Component {
     this.renderSubForms();
   }
 
+  @action moveUp(subForm) {
+    this.changeItemOrder(subForm, true);
+  }
+
+  @action moveDown(subForm) {
+    this.changeItemOrder(subForm, false);
+  }
+
+  changeItemOrder(subform, shouldMoveUp) {
```

Comment:
>Not mandatory
Could we make an "enum" (constant) for this instead of a boolean?
```js
const DIRECTION = {
   UP: 'up',
   DOWN: 'down
} 
```

So it will become 
```js
this.changeItemOrder(subForm, DIRECTION.UP)
```

Reply:
Yea, I figured it was overkill since there were only 2 options, but it makes it bit more readable so I added it anyways. 👍